packages/node_modules/@webex/plugin-meetings/src/statsAnalyzer/index.js

```diff
@@ -0,0 +1,373 @@
+import EventsScope from '../common/events/events-scope';
+import {
+  DEFAULT_GET_STATS_FILTER, EVENT_TRIGGERS, PEER_CONNECTION_STATE, STATS
+} from '../constants';
+import LoggerProxy from '../common/logs/logger-proxy';
+
+import defaultStats from './global';
+
+/**
+  * Meeting High Packet Loss Event
+  * Emitted when ever there is high packet loss detected
+  * @event meeting:highPacketLoss
+  * @type {Object}
+  * @property {String} mediaType Type of media experiencing packet loss
+  * @property {Number} packetLossRatio current packet loss ratio
+  * @memberof StatsAnalyzer
+  */
+
+/**
+ * Stats Analyzer class that will emit events based on detected quality
+ *
+ * @export
+ * @class StatsAnalyzer
+ * @extends {EventsScope}
+ */
+export default class StatsAnalyzer extends EventsScope {
```

Comment:
This class has a lot of magic, do we have a JIRA for adding tests to this?

Reply:
Good call: https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-147674